moped-database/migrations/1655490225419_update_proj_partner_filter/up.sql

```diff
@@ -0,0 +1,149 @@
+-- public.project_list_view source
```

Comment:
No problem with this here, especially in a migration that isn't intended to be human readable, but just out of curiosity - what does this comment indicate?

Reply:
It is a Hasura generated thing so it probably can go! Just an oversight so I'm glad that you pointed it out.

Reply:
Actually, I just noticed it is in the version controlled file. I'll get that out of there so we don't keep pushing it into migrations. Thanks!